sql-plugin/src/main/scala/com/nvidia/spark/rapids/conditionalExpressions.scala

```diff
@@ -93,7 +129,8 @@ case class GpuIf(
 
 case class GpuCaseWhen(
     branches: Seq[(Expression, Expression)],
-    elseValue: Option[Expression] = None) extends GpuConditionalExpression with Serializable {
+    elseValue: Option[Expression] = None,
+    optAllTrueFalse: Boolean) extends GpuConditionalExpression with Serializable {
```

Comment:
Non-default arguments should not appear after default arguments.

Reply:
I decided to remove the config, so this is out of date.